tests/Integration/ScriptsTest.php

```diff
@@ -51,40 +49,23 @@ public function set_up(): void {
 	 * @uses \Parsely\Parsely::api_key_is_set
 	 * @uses \Parsely\Parsely::get_options
 	 * @uses \Parsely\Parsely::update_metadata_endpoint
-	 * @group insert-js
+	 * @group enqueue-js
 	 */
 	public function test_parsely_register_scripts(): void {
-		ob_start();
-		$post_array = $this->create_test_post_array();
-		$post       = $this->factory->post->create( $post_array );
-		$this->go_to( '/?p=' . $post );
 		self::$scripts->register_scripts();
```

Comment:
Can you add a couple of `assert_script_statuses()` to show that both scripts are not even `registered`, before calling this `register_scripts()`?

Reply:
Sure!